core/src/test/java/nl/nn/adapterframework/jdbc/DirectQuerySenderTest.java

```diff
@@ -0,0 +1,51 @@
+package nl.nn.adapterframework.jdbc;
+
+import static org.junit.Assert.assertEquals;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import nl.nn.adapterframework.jdbc.dbms.H2DbmsSupport;
+import nl.nn.adapterframework.senders.SenderTestBase;
+import nl.nn.adapterframework.util.AppConstants;
+
+public class DirectQuerySenderTest extends SenderTestBase<DirectQuerySender> {
+	private static final String H2_CONNECTION_STRING = "jdbc:h2:mem:test";
+
+	private Connection connection;
+
+	@Before
+	public void setUp() throws Exception {
+		AppConstants.getInstance().setProperty("jdbc.convertQueriesFrom", "Oracle");
+		startDatabase();
+		super.setUp();
+	}
+
+	public void startDatabase() throws SQLException {
+		connection = DriverManager.getConnection(H2_CONNECTION_STRING);
+		connection.createStatement().execute("CREATE TABLE TEMP(TKEY INT PRIMARY KEY, TVARCHAR VARCHAR(100), TVARCHAR2 VARCHAR(100), TINT INT, TDATETIME DATETIME)");
+	}
+
+	@Override
+	public DirectQuerySender createSender() throws Exception {
+		DirectQuerySender dqs = new DirectQuerySender();
+		dqs.setDbmsSupport(new H2DbmsSupport());
+		return dqs;
+	}
+
+	@Test
+	public void testConvertQuery() throws JdbcException, SQLException {
```

Comment:
Wat test je hier precies? Het lijkt er op dat het voornamelijk 'splitQuery()' is.
Het lijkt me goed om die concreet apart te testen, zonder dat er iets geconverteerd wordt.

Reply:
https://github.com/ibissource/iaf/pull/251/commits/7f1e58ce633afc3e476a8c96ff502d5b12ff1368